examples/04-Exporting-ranking-models.ipynb

```diff
@@ -70,9 +70,9 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        train, valid = generate_data("aliccp-raw", int(NUM_ROWS), set_sizes=(0.7, 0.3))
</pre><p>is there an <code>else</code>  statement? what will happen if it is not True?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>The if statement writes data into the DATA_FOLDER. If it is not True, then it assumes there is already data in the folder (the raw data)</p>